src/pabot/pabotlib.py

```diff
@@ -154,10 +154,10 @@ def add_value_to_set(self, name, content):
             content[self._TAGS_KEY] = []
         self._values[name] = content
 
-    def import_shared_library(self, name):  # type: (str) -> int
+    def import_shared_library(self, name, args=None):  # type: (str, Iterable[Any]|None) -> int
```

Comment:
I made args a kwarg here (ironically) so as to not interfere with any other potential uses of this keyword. If it's only the SharedLibrary class that will ever call this though, it can probably just be an arg.